--- etc/udev/rules.d/60-persistent-storage.rules        2015-03-18 20:00:03.947462040 +0000
+++ etc/udev/rules.d/60-persistent-storage.rules        2015-03-18 19:59:42.043458703 +0000
@@ -38,8 +38,9 @@
 KERNEL!="sr*", OPTIONS+="watch"

 # by-label/by-uuid links (filesystem metadata)
+IMPORT{program}="/sbin/blkid -o udev -p $tempnode"
 ENV{ID_FS_USAGE}=="filesystem|other|crypto", ENV{ID_FS_UUID_ENC}=="?*", SYMLINK+="disk/by-uuid/$env{ID_FS_UUID_ENC}"
-ENV{ID_FS_USAGE}=="filesystem|other", ENV{ID_FS_LABEL_ENC}=="?*", SYMLINK+="disk/by-label/$env{ID_FS_LABEL_ENC}"
+ ENV{ID_FS_USAGE}=="filesystem|other", ENV{ID_FS_LABEL_ENC}=="?*", SYMLINK+="disk/by-label/$env{ID_FS_LABEL_ENC}", RUN+="/bin/mknod /mnt/.tether/volumes/$env{ID_FS_LABEL_ENC} b $major $minor"

 # by-id (World Wide Name)
 ENV{DEVTYPE}=="disk", ENV{ID_WWN_WITH_EXTENSION}=="?*", SYMLINK+="disk/by-id/wwn-$env{ID_WWN_WITH_EXTENSION}"

